.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: brettt89/silverstripe-web
+      - image: mattrayner/lamp:latest-1804
```

Comment:
Why the switch of docker image?

Reply:
Beacuse that image is like php 5.6 or something.

Reply:
Brett also has a 7.x image now, brettt89/silverstripe-web:{php-version}